version-specific/mqt-compat/.pre-commit-config.yaml

```diff
@@ -0,0 +1,53 @@
+exclude: |
+  (?x)
+  # NOT INSTALLABLE ADDONS
+  # END NOT INSTALLABLE ADDONS
+  # Files and folders generated by bots, to avoid loops
+  ^setup/|/static/description/index\.html$|
+  # We don't want to mess with tool-generated files
+  .svg$|/tests/([^/]+/)?cassettes/|^.copier-answers.yml$|^.github/|
+  # Maybe reactivate this when all README files include prettier ignore tags?
+  ^README\.md$|
+  # Library files can have extraneous formatting (even minimized)
+  /static/(src/)?lib/|
+  # Repos using Sphinx to generate docs don't need prettying
+  ^docs/_templates/.*\.html$|
+  # You don't usually want a bot to modify your legal texts
+  (LICENSE.*|COPYING.*)
+default_language_version:
+  python: python3
+  node: "14.13.0"
+repos:
+  - repo: https://github.com/oca/maintainer-tools
+    rev: ab1d7f6
+    hooks:
+      # update the NOT INSTALLABLE ADDONS section above
+      - id: oca-update-pre-commit-excluded-addons
+      - id: oca-fix-manifest-website
+        args: ["https://github.com/OCA/{{ repo_slug }}"]
+  - repo: https://github.com/acsone/setuptools-odoo
```

Comment:
A lot of checks are missing here compared to 13.0:

- forbidden-files
- autoflake
- black
- prettier
- pre-commit-hooks
- pyupgrade
- isort

Some of these exclusions seem intentional, but I'm not sure all of them are.

Reply:
That is intentional. My goal is to imitate what oca/mqt does, so we create as little disruption as possible when we are going to apply this to 10,11,12 in order to shut down travis.